### PR TITLE
[6.3] FIX  CLI HostSubscriptionTestCase

### PR DESCRIPTION
**Pull request by ldjebran**
```
(sat-6.3.0) dlezz@elysion:~/projects/robottelo-fork$ py.test -v tests/foreman/cli/test_host.py::HostSubscriptionTestCase================================================= test session starts ==================================================
platform linux2 -- Python 2.7.13, pytest-3.1.3, py-1.4.34, pluggy-0.4.0 -- /home/dlezz/.pyenv/versions/sat-6.3.0/bin/python2.7
cachedir: .cache
rootdir: /home/dlezz/projects/robottelo-fork, inifile:
plugins: xdist-1.15.0, services-1.2.1, mock-1.6.2, cov-2.4.0
collected 8 items 
2017-08-17 09:08:31 - conftest - DEBUG - Found WONTFIX in decorated tests ['1147100', '1269208', '1269196', '1378009', '1245334', '1217635', '1226425', '1156555', '1311113', '1199150', '1238247', '1204686', '1267224', '1221971', '1103157', '1230902', '1230865', '1214312', '1079482']

2017-08-17 09:08:31 - conftest - DEBUG - Collected 8 test cases


tests/foreman/cli/test_host.py::HostSubscriptionTestCase::test_negative_without_attach PASSED
tests/foreman/cli/test_host.py::HostSubscriptionTestCase::test_negative_without_attach_with_lce PASSED
tests/foreman/cli/test_host.py::HostSubscriptionTestCase::test_positive_attach <- ../../.pyenv/versions/sat-6.3.0/lib/python2.7/site-packages/robozilla/decorators/__init__.py PASSED
tests/foreman/cli/test_host.py::HostSubscriptionTestCase::test_positive_attach_with_lce <- ../../.pyenv/versions/sat-6.3.0/lib/python2.7/site-packages/robozilla/decorators/__init__.py PASSED
tests/foreman/cli/test_host.py::HostSubscriptionTestCase::test_positive_auto_attach PASSED
tests/foreman/cli/test_host.py::HostSubscriptionTestCase::test_positive_register PASSED
tests/foreman/cli/test_host.py::HostSubscriptionTestCase::test_positive_remove PASSED
tests/foreman/cli/test_host.py::HostSubscriptionTestCase::test_positive_unregister PASSED

================================================== 0 tests deselected ==================================================
============================================= 8 passed in 1769.61 seconds ==============================================
```

### CODE REVIEW

File: tests/foreman/cli/test_host.py [hunk 1]
```diff
@@ -2207,7 +2207,7 @@ def setUpClass(cls):
             u'lifecycle-environment-id': cls.env['id'],
             u'activationkey-id': cls.activation_key['id'],
             u'subscription': cls.subscription_name,
-        })
+        }, force_use_cdn=True)
```
Comment (abalakh) [inline tests/foreman/cli/test_host.py]: Pls explain why is this necessary. We'll better avoid using cdn version where it's possible as we have to test the latest changes which will be delivered altogether with satellite to avoid regressions.

Comment (ldjebran) [inline tests/foreman/cli/test_host.py]: @abalakh this test use CDN to test subscriptions, nothing harmfull to use CDN here, and has to install some packages and to assert that that packages was installed.

Comment (ldjebran) [inline tests/foreman/cli/test_host.py]: @abalakh we are not testing the packages them selves, or what can do, only that that repos can be enabled and one of the packages from that installed.

Comment (ldjebran) [inline tests/foreman/cli/test_host.py]: we have already fixed that on 6.2 , but we missed to do it in 6.3 
https://github.com/SatelliteQE/robottelo/blob/6.2.z/tests/foreman/cli/test_host.py#L1424
